internal/pkg/manifest/validate.go

```diff
@@ -1498,6 +1498,15 @@ func (s SidecarConfig) validate() error {
 	}
 	return s.ImageOverride.validate()
 }
+func (s SidecarConfig) validateSideCarImageFields() error {
```

Comment:
nit, for consistency with struct
```suggestion
func (s SidecarConfig) validateSidecarImageFields() error {
```